### PR TITLE
legacy: OAI Harvest admin input stripping

### PR DESCRIPTION
**Pull request by jalavik**
* Strips the input of certain fields when adding and editing
  OAI harvest source configurations.

Signed-off-by: Jan Aage Lavik <jan.age.lavik@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
@jalavik why `v2.2.0`?

**Comment by jalavik** [comment]
Err, I meant 2.1.0

File: invenio/legacy/oaiharvest/admin.py [hunk 1]
```diff
@@ -243,14 +243,14 @@ def perform_request_editsource(oai_src_id=None, oai_src_name='',
                 oai_src_post = []
 
             oai_src = OaiHARVEST.query.get(oai_src_id)
-            oai_src.setspecs = " ".join(oai_src_sets)
-            oai_src.baseurl = oai_src_baseurl
-            oai_src.name = oai_src_name
-            oai_src.metadataprefix = oai_src_prefix
-            oai_src.postprocess = "-".join(oai_src_post)
+            oai_src.setspecs = " ".join(oai_src_sets).strip()
```
Comment (jirikuncar) [inline invenio/legacy/oaiharvest/admin.py]: maybe it would be better to create validators/setters for the fields in model, wdyt?

**Comment by jirikuncar** [comment]
:shipit:

**Pull request merged**